### ISSUE TITLE
[Enhancement]: Better recovery of attributes for Animator Parameters and BlensShapes in AnimationClips

### ISSUE THREAD

**Issue by Zbuffers** [AssetRipper/AssetRipper#1009]
### Describe the new feature or enhancement

Hi,

I sometimes encounter cases  where animationClips don't have the proper attributes for Animator parameters and Skinned Mesh BlendShapes.

In order to recover those, I just used a custom tool to list/find the CRC32 hash of those attributes (see my post in #954)

In the following example, 
typetree_2757744376 is in fact the CRC32 hash of one of the parameters of the Animator that references this animationClip:
CRC32           -> animator parameter name
2757744376  -> IKWeight

```
    attribute: typetree_2757744376 
    path:
    classID: 95
```
Should be:
```
    attribute: IKWeight
    path:
    classID: 95
```
Where IKWeight is an animator parameter name.

And same goes for Blenshapes:
CRC32           -> Mesh Blend Shape name
3439576345 -> Facial_Blendshape.BrowsUp

```
    attribute: blendShape.3439576345
    path: Geo/Head
    classID: 137
```
should be 

```
    attribute: blendShape.Facial_Blendshape.BrowsUp
    path: Geo/Head
    classID: 137
```

I hope i am being clear enough ;)
Kind regards,
Z

**Comment by ds5678** [comment]
@mitakabg20 I saw your comment before you deleted it. Let me know if the above commit improved things for you.

**Comment by mitakabg20** [comment]
Thank you! I deleted it because I used the tool in #954 . I will try again later.

**Comment by mitakabg20** [comment]
Decompiled the game again, and it looks like there's still a problem with some animations<img width="823" height="186" alt="Image" src="https://github.com/user-attachments/assets/841fb63c-9850-4949-9b53-5135173ffa92" />

**Comment by ds5678** [comment]
@mitakabg20 are you on the latest commit? Did my changes cause a reduction in the number of issues?

**Comment by mitakabg20** [comment]
Yep, I downloaded a new version, and even built the fork I cloned of AR. But the issues are same as before.

**Comment by ds5678** [comment]
What about now?

**Comment by mitakabg20** [comment]
Yes! Thank you, its working properly now! Only the names are messed up but the blend shapes seem to be working. Thank you so much! :D

<img width="680" height="207" alt="Image" src="https://github.com/user-attachments/assets/30557a0f-4481-45c9-b4e7-75edef666606" />
